DuckDuckGo/PrivacyProtectionController.swift

```diff
@@ -28,22 +28,10 @@ protocol PrivacyProtectionDelegate: class {
 
 }
 
-class PrivacyProtectionController: UIViewController {
+class PrivacyProtectionController: ThemableNavigationController {
 
-    override var preferredStatusBarStyle: UIStatusBarStyle {
-        return ThemeManager.shared.currentTheme.statusBarStyle
-    }
-
-    @IBOutlet weak var contentContainer: UIView!
-    @IBOutlet weak var omniBarContainer: UIView!
-    @IBOutlet weak var statusBarBackground: UIView!
-    @IBOutlet weak var headerConstraint: NSLayoutConstraint!
-
-    weak var delegate: PrivacyProtectionDelegate?
-
-    weak var embeddedController: UINavigationController!
+    weak var privateProtectionDelegate: PrivacyProtectionDelegate?
```

Comment:
This is playing tricks with my mind (looks like `var private protectionDelegate` ) could it be named `var privacyProtectionDelegate` ? :)